src/frontend/src/pages/TeamsPage/TeamSpecificPage.tsx

```diff
@@ -42,7 +42,16 @@ const TeamSpecificPage: React.FC = () => {
               </Grid>
             </Grid>
           </PageBlock>
-          <PageBlock title={'Active Projects'}></PageBlock>
+          <PageBlock title={'Active Projects'}>
+            <Grid container spacing={2}>
+              {data.projects.map((project) => (
```

Comment:
like i said, lets have it send all projects and filter by status by updating the project preview interface to have the status